e2e-tests/tests/translation-version-name-appears-in-title.spec.ts

```diff
@@ -0,0 +1,80 @@
+import {test, expect} from '@playwright/test';
+import {goToPageWithLang, changeLanguageOfText} from "../utils";
+import {LANGUAGES, SOURCE_LANGUAGES} from '../globals'
+
+[
+    // Hebrew Interface and Hebrew Source
+    {interfaceLanguage: 'Hebrew', interfaceLanguageToggle: LANGUAGES.HE, 
+        sourceLanguage: 'Hebrew', sourceLanguageToggle: SOURCE_LANGUAGES.HE,
+        translations: 'תרגומים', select: 'בחירה', currentlySelected: 'נוכחי'}, 
+
+    // Hebrew Interface and Bilingual Source
+    {interfaceLanguage: 'Hebrew', interfaceLanguageToggle: LANGUAGES.HE, 
+        sourceLanguage: 'Bilingual', sourceLanguageToggle: SOURCE_LANGUAGES.BI,
+        translations: 'תרגומים', select: 'בחירה', currentlySelected: 'נוכחי'}, 
+    
+    // Hebrew Interface and English Source
+    {interfaceLanguage: 'Hebrew', interfaceLanguageToggle: LANGUAGES.HE, 
+        sourceLanguage: 'English', sourceLanguageToggle: SOURCE_LANGUAGES.EN,
+        translations: 'תרגומים', select: 'בחירה', currentlySelected: 'נוכחי'}, 
+
+    // English Interface and English Source
+    {interfaceLanguage: 'English', interfaceLanguageToggle: LANGUAGES.EN, 
+        sourceLanguage: 'English', sourceLanguageToggle: SOURCE_LANGUAGES.EN,
+        translations: 'Translations', select: 'Select', currentlySelected: 'Currently Selected'},
+
+    // English Interface and English Source
+    {interfaceLanguage: 'English', interfaceLanguageToggle: LANGUAGES.EN, 
+        sourceLanguage: 'Bilingual', sourceLanguageToggle: SOURCE_LANGUAGES.BI,
+        translations: 'Translations', select: 'Select', currentlySelected: 'Currently Selected'},
+    
+    // English Interface and Hebrew Source
+    {interfaceLanguage: 'English', interfaceLanguageToggle: LANGUAGES.EN, 
+        sourceLanguage: 'Hebrew', sourceLanguageToggle: SOURCE_LANGUAGES.HE,
+        translations: 'Translations', select: 'Select', currentlySelected: 'Currently Selected'}
+
+].forEach(({interfaceLanguage, interfaceLanguageToggle, sourceLanguage, sourceLanguageToggle, translations, currentlySelected, select}) => {
+    test(`${interfaceLanguage} - translation name appears in title for ${sourceLanguage} source text`, async ({ context }) => {
+         // Navigate to Bereshit in specified Interface Language
+         const page = await goToPageWithLang(context,'/Genesis.1', `${interfaceLanguageToggle}`)
+
+         // Change the Source Language of the text
+         await changeLanguageOfText(page, `${sourceLanguageToggle}`)
+ 
+         // Retain the translation name locator
+         const translationNameInTitle = page.locator('span.readerTextVersion')
+ 
+         // Navigate to the Translations sidebar by clicking on the text title
+         //Clicks on בראשית א׳ / Genesis I
+         await page.locator('h1').click()
```

Comment:
Same comment throughout this file regarding locator use - I didn't comment in each location, just a general curiosity to see if it's possible to rewrite any of these.

Reply:
Hmm... I can change it to `await page.getByRole('heading').first().click()`. This still works, and it is using best practices.  What do you think?

Reply:
I think that would be great, thank you for the suggestion!